Vagrantfile

```diff
@@ -73,6 +73,7 @@ Vagrant.configure(2) do |config|
         vm_config.vm.provision "shell", inline: "yum install -y centos-release-scl"
         vm_config.vm.provision "shell", inline: "yum install -y java-1.8.0-openjdk unzip git rh-ruby22-rubygem-rake"
         vm_config.vm.provision "shell", inline: "echo 'source /opt/rh/rh-ruby22/enable' > /etc/profile.d/ruby-22.sh"
+        vm_config.vm.provision "shell", inline: "nameserver 8.8.8.8 >> /etc/resolv.conf"
```

Comment:
should it be `echo nameserver 8.8.8.8 >> /etc/resolv.conf` ?

Reply:
Oops. Sorry. Fixed now.